tests/deprecated_classes.py

```diff
@@ -1518,6 +1518,16 @@
         'airflow.providers.amazon.aws.operators.s3_file_transform.S3FileTransformOperator',
         'airflow.operators.s3_file_transform_operator.S3FileTransformOperator',
     ),
+    (
+        'airflow.providers.amazon.aws.operators.step_function.StepFunctionStartExecutionOperator',
+        'airflow.providers.amazon.aws.operators.s3_file_transform_operator.'
+        'StepFunctionStartExecutionOperator',
```

Comment:
what is this extra entry?

Reply:
It's how long string have been handled in this file

https://github.com/apache/airflow/blob/main/tests/deprecated_classes.py#L474